README.md

```diff
@@ -155,7 +155,7 @@ Slotted machines are determined based on the id of the invoked/spawned machine.
 `singleSlot` accepts the name of the slot as the first argument and returns an object with a method `getId()` that returns the id of the slot.  
 `multiSlot` accepts the name of the slot and returns an object with a method `getId(id: string)` that returns the id of the slot
 
-You should always use the `getId` methods when invoking/spawning something into a slot. Each slot the machine has must be represented by a call to `singleSlot` or `multiSlot` and stored into an array of slots. These slots must be passed to the `buildView` and `buildXStateTreeMachine` functions.
+You should always use the `getId` methods when invoking/spawning something into a slot. Each slot the machine has must be represented by a call to `singleSlot` or `multiSlot` and stored into an array of slots. These slots must be passed to the `createXStateTreeMachine` function.
```

Comment:
The codesandbox example also probably needs to be updated to replace the deprecated API methods.

Reply:
Had a go at updating the codesandbox example but no luck. I think `createXStateTreeMachine` builder only works with xstate typegen enabled machines which is kind of a PITA in a codesandbox. For some reason typescript is only allowing the selector function to return the machines context, which is... incorrect.